plugins/sigma.plugins.design/sigma.plugins.design.js

```diff
@@ -1001,6 +1001,78 @@
     };
 
     /**
+     * Deletes styles from an element according to the specified options.
+     * If the only the nodeId is specified, this will delete the 'size', 'color', 'type', 'icon' and
+     * 'image'
+     *
+     * @param {String} target - The data target. Available values: "nodes", "edges".
+     * @param {Number|String} id - id of the Element to update
+     * @param {Object} [key] - property key to delete styles from.
+     */
+    this.deleteElementStyles = function(target, id, key){
+
+      if (id === undefined || id === null){
```

Comment:
`id === undefined || id === null` is equivalent to `id == null`, because `undefined == null` so you can check directly `id == null`